test/NuGet.Core.Tests/NuGet.Commands.Test/BuildAssetsUtilsTests.cs

```diff
@@ -745,7 +745,7 @@ public async Task BuildAssetsUtils_GeneratePathProperty()
 
                 Assert.Equal($" '$({expectedProperty.Name.LocalName})' == '' ", expectedProperty.Attribute("Condition")?.Value);
 
-                Assert.Equal(packageDirectory.FullName, expectedProperty?.Value, ignoreCase: true);
+                Assert.Equal($@"{packageDirectory.FullName}{Path.DirectorySeparatorChar}", expectedProperty?.Value, ignoreCase: true);
```

Comment:
Do we need string verbatim `@` here?
```suggestion
                Assert.Equal($"{packageDirectory.FullName}{Path.DirectorySeparatorChar}", expectedProperty?.Value, ignoreCase: true);
```

Reply:
It's not blocking.

Reply:
Good catch, I originally placed a backslash in there since MSBuild will normalize paths anyway but this is a property value so it wasn't technically correct.  I used `Path.DirectorySeparatorChar` instead but forgot to remove the verbatim indicator.